package.json

```diff
@@ -28,7 +28,6 @@
     "eslint": "^4.15.0",
     "eslint-config-brightspace": "^0.4.0",
     "eslint-plugin-html": "^4.0.1",
-    "frau-ci": "^1",
```

Comment:
Nice, I think we missed this in `button-group`

Reply:
Yep -- just fixed it directly there thx.